### PR TITLE
Fix/asset fingerprint internal impl

### PR DESCRIPTION
**Pull request by mirceahasegan**
# Context

It is simple enough to implement this in the SDK

### CODE REVIEW

**Review by rhyslbw — CHANGES_REQUESTED** [review]
Looks good @mirceahasegan

File: packages/core/test/Cardano/types/Asset.test.ts [hunk 1]
```diff
@@ -76,6 +79,47 @@ describe('Cardano/types/Asset', () => {
       const policyId = PolicyId('659f2917fb63f12b33667463ee575eeac1845bbc736b9c0bbc40ba82');
       const assetName = AssetName('54534c41');
       expect(AssetFingerprint.fromParts(policyId, assetName)).toEqual('asset1rqluyux4nxv6kjashz626c8usp8g88unmqwnyh');
```
[the remark this answers is not in the document]
Comment (mirceahasegan) [reply packages/core/test/Cardano/types/Asset.test.ts]: I updated the test name to `creates expected fingerprint from cip14 test vectors` and removed the test that was not part of the test vectors: [4add154](https://github.com/input-output-hk/cardano-js-sdk/pull/621/commits/4add154f36cf2083ae99d4806f2f5f44ecbc6485)

File: packages/crypto/src/index.ts [hunk 2]
```diff
@@ -1,3 +1,5 @@
+import blake2b from 'blake2b';
+export { blake2b };
```
[the remark this answers is not in the document]
Comment (mkazlauskas) [reply packages/crypto/src/index.ts:2]: Let's use that then

**Review by mkazlauskas — APPROVED** [review]
Nice work! :rocket:

**Review by rhyslbw — CHANGES_REQUESTED** [review]
Looks good @mirceahasegan. Probably better to make [this](https://github.com/input-output-hk/cardano-js-sdk/pull/621/commits/4add154f36cf2083ae99d4806f2f5f44ecbc6485) a refactor type since it's not a bug on our side, and the commit message reads like the former.

**Review by rhyslbw — CHANGES_REQUESTED** [review]
Just realised the [package is still listed as a dependency](https://github.com/input-output-hk/cardano-js-sdk/blob/276f63deb3ab81e5eeda75877ffbeaa57c94771f/packages/core/package.json#L79)

**Comment by mirceahasegan** [comment]
> Just realised the [package is still listed as a dependency](https://github.com/input-output-hk/cardano-js-sdk/blob/276f63deb3ab81e5eeda75877ffbeaa57c94771f/packages/core/package.json#L79)

Great catch!

**Comment by mirceahasegan** [comment]
@rhyslbw , I removed cip-14 package and force pushed

**Review by mkazlauskas — APPROVED** [review]

**Review by rhyslbw — APPROVED** [review]
:tada:

**Review by rhyslbw — APPROVED** [review]

**Pull request merged**